### PR TITLE
Gamemode code improvement

### PR DESCRIPTION
**Pull request by ghost**
:cl: Xhuis
add: Added communication intercept texts for gamemodes that were missing them.
tweak: Changed all intercept texts and improved grammar for a lot of key game code.
/:cl:
- Improved the grammar in some key code like pre-game lobby notices and the communication intercept message.
- Added communication intercept text for clockwork cult and tweaked existing ones. Now five gamemodes will be in the intercept, with one of those being the correct one.
- Removed the unused faction datums pending implementation in the future.
- Removes the unusued code for pointing out who antagonists are in the intercept.
- Unified `announce()` for gamemodes by implementing two new variables that are used in the proc.

This is merge-ready and required before I can start on datum antags.

### CODE REVIEW

**Comment by Bawhoppen** [comment]
on par with datum antags

File: code/game/gamemodes/factions.dm [hunk 1]
```diff
@@ -1,176 +1 @@
-
-// Normal factions:
-
-/datum/faction
```
Comment (ChangelingRain) [inline code/game/gamemodes/factions.dm]: this should be kept as a curiosity really

File: code/game/gamemodes/intercept_report.dm [hunk 2]
```diff
@@ -1,281 +1,58 @@
...
-	src.text += "Additionally, the report indicates a [prob_right_dude]% chance that [changeling_name] may have been in contact with the lifeform at a recent social gathering.  "
-	*/
-	src.text += "These lifeforms are associated with the <B><U>[orgname1] [orgname2]</U></B> and may be attempting to acquire sensitive materials on their behalf.  "
-	src.text += "Please take care not to alarm the crew, as <B><U>[cname]</U></B> may take advantage of a panic situation. Remember, they can be anybody, suspect everybody!"
-	src.text += "<BR><HR>"
+			text += "Although more specific threats are commonplace, you should always remain vigilant for Syndicate agents aboard your station. Syndicate communications have implied that many \
+			Nanotrasen employees are Syndicate agents with hidden memories that may be activated at a moment's notice, so it's possible that these agents might not even know their positions."
+		if("wizard")
+			text += "A dangerous Wizards' Federation individual by the name of Old Man Henderson has recently escaped confinement from an unlisted prison facility. This man \
```
Comment (ChangelingRain) [inline code/game/gamemodes/intercept_report.dm]: one henderson of station destruction

File: code/game/gamemodes/factions.dm (hunk 1 above)
Comment (Xhuis) [inline code/game/gamemodes/factions.dm]: I'm going to re-add them.

**Comment by Shadowlight213** [comment]
Why remove random name generation? Especially if it's going to be replaced with player references.

**Comment by Jalleo** [comment]
That would be a undocumented change.

**Comment by KorPhaeron** [comment]
>rewriting 6+ year old fluff lines

I hate change

**Comment by Xhuis** [comment]
@KorPhaeron I think. That Bad Capital And Punctuation Should. Be removed!

**Comment by Xhuis** [comment]
In other news, this should be merge-ready unless anyone has some further improvements they want me to do.

**Comment by Jalleo** [comment]
@Xhuis wait wasnt the meta of definitly having the correct game mode on the report removed?

**Comment by Bawhoppen** [comment]
@Jalleo it was many years ago

**Comment by Jalleo** [comment]
Then we have a problem
Added communication intercept text for clockwork cult and tweaked existing ones. Now five gamemodes will be in the intercept, with one of those being the correct one.

**Comment by Bawhoppen** [comment]
What's the problem?

File: code/game/gamemodes/factions.dm (hunk 1 above)
Comment (Bawhoppen) [inline code/game/gamemodes/factions.dm]: these factions are all fake lore please keep them removed

**Comment by Xhuis** [comment]
@Jalleo @Bawhoppen Baw, you're misinformed and incorrect. If you took the time to read the code, you'd see that the old code still had the correct gamemode.

**Comment by Bawhoppen** [comment]
no it just had memes this is an outrage

**Comment by Jalleo** [comment]
Right okay good to know

**Pull request merged**